contrib/hadoop-store-builder/src/java/voldemort/store/readonly/mr/azkaban/VoldemortBuildAndPushJob.java

```diff
@@ -719,7 +722,7 @@ public String runBuildStore(Props props, String url) throws Exception {
 
     public void runPushStore(Props props, String url, String dataDir) throws Exception {
         // For backwards compatibility http timeout = admin timeout
-        int httpTimeoutMs = 1000 * props.getInt(PUSH_HTTP_TIMEOUT_SECONDS, 24 * 60 * 60);
+        int httpTimeoutMs = 1000 * props.getInt(PUSH_HTTP_TIMEOUT_SECONDS, 60);
```

Comment:
This will break the `AdminStoreSwapper`, because it also uses the same parameter for the total fetch time. For total fetch time, 24 hours is not necessarily unreasonable (unlimited may be fine too). The problem is that we use the same config parameter for both of these settings. We should either stop using this parameter in the `AdminStoreSwapper` (and then I guess we may want to default to -1 for unlimited fetch time) or we could create a new setting so that these two are controlled separately. I'll leave it up to you which of these two options you prefer, but this cannot be merged in without either one of them.